build.gradle

```diff
@@ -1,8 +1,12 @@
+import uk.gov.hmrc.components.buildsrc.Kotlin
+import uk.gov.hmrc.components.buildsrc.Gradle
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
 
     ext {
         compose_version = '1.3.3'
```

Comment:
i think we can update the `compose_version` to `compose_version = '1.4.1'`

ref: https://github.com/hmrc/android-components/commit/c92c7ed5433de3a96376af359cb14626ef474332

please check this branch and few more libraries are available to update if needed